packages/verification-vue-component/src/components/Verification/items/cards/ContactCard.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <BaseCard class="help-card">
+        <template #header>
+            <div class="title">
+                <span class="help-card-title">{{ _$t('verification.card.contact.title') }}</span>
+            </div>
+        </template>
+        <template #body>
+            <form id="contact-form" @submit="checkForm" method="POST">
+                <p class="email-field" :class="{error: errors.email}">
```

Comment:
I would use div's instead of p's, it's really a paragraph.

Reply:
adjusted it